.github/workflows/check_code_style.yml

```
@@ -0,0 +1,23 @@
+name: Check Code Style
+
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    branches:
+      - "main" 
+
+jobs:
+  check_code_style:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Initialize and update submodule
+        run: git submodule update --init
+
+      # Call the composite action to check files
+      # for correct code style. This action (action.yml)
+      # is in `3rdparty/dev-tools` submodule.
+      - uses: 3rdparty/dev-tools@main
```

This is unfortunate naming. It looks like though that we _can_ have multiple actions per repository, see [here](https://github.community/t/possible-to-have-multiple-actions-in-a-single-repo/159823). Then we can put `action.yml` into the `check-code-style` directory of `dev-tools` and do:
```suggestion
      - uses: 3rdparty/dev-tools/check-code-style@main
```

---

This [PR](https://github.com/3rdparty/dev-tools/pull/14) is waiting for you @benh !

---

Ok approved! Now you can update and commit!